aviatrix_ha.py

```diff
@@ -504,6 +506,25 @@ def verify_backup_file(controller_instanceobj):
         return True, s3_file
 
 
+def support_upgrade_to_build(ip_addr, cid):
+    """ Check if the version supports upgrade to build """
+    print("Checking if upgrade to build is suppported")
+    base_url = "https://" + ip_addr + "/v1/api"
+    post_data = {"CID": cid,
+                 "action": "get_feature_info"}
+    try:
+        response = requests.post(base_url, data=post_data, verify=False)
+        print(response.json())
```

Comment:
json decode could fail. you could do json.loads(response.content) and print response.content if it fails with ValueError. 
Also check for return Value. if return is False print the reason.

Reply:
Changed to use response.content and handled exceptions